packages/binutils/binutils.spec

```diff
@@ -63,6 +63,10 @@ Requires: %{name}
 %exclude %{_cross_bindir}/readelf
 %exclude %{_cross_bindir}/size
 %exclude %{_cross_bindir}/strings
+%exclude %{_cross_bindir}/gp-*
+%exclude %{_cross_bindir}/gprofng*
+%exclude %{_cross_libdir}/gprofng/*
```

Comment:
I'd prefer to disable this feature in the initial `configure` invocation, via `--disable-gprofng`. Then you won't need these additional excludes.